include/flashinfer/attention/cascade.cuh

```diff
@@ -152,23 +152,40 @@ __global__ void MergeStatesKernel(DTypeIn* __restrict__ V, float* __restrict__ S
   uint32_t tx = threadIdx.x, ty = threadIdx.y;
   uint32_t pos = blockIdx.x;
   uint32_t head_idx = ty;
-  state_t<vec_size> st;
 
-  vec_t<float, vec_size> v_merged_vec;
-  v_merged_vec.fill(0.f);
+  if (num_index_sets > 1) {
```

Comment:
wouldn't it be cleaner to just early return instead of indenting everything?

Reply:
yes you are right, thanks for your suggestion.